README.md

```diff
@@ -57,6 +57,31 @@ You'll also need a GitHub token and a config file. (Keep reading for more info o
 1. Store your GitHub token in your system's environment by running `export MARINER_GITHUB_TOKEN={Insert your GitHub token here}`. You will either have to do this once each time you restart your system, or else configure your system to do so automatically.  
 1. Finally, run the application to find open issues in your dependencies, using the command `node index.js`.
 
+### Optional: Generating Markdown
+- You can generate markdown for use in Confluence/jira
+- The generateConfluenceMarkdown.ts file is inside the `src/Utilities` folder
```

Comment:
We won't need to tell users this, because they will just be able to import the function via mariner.